integration/go.mod

```diff
@@ -25,7 +25,7 @@ require (
 	github.com/onflow/flow/protobuf/go/flow v0.3.1
 	github.com/plus3it/gorecurcopy v0.0.1
 	github.com/prometheus/client_golang v1.12.1
-	github.com/rs/zerolog v1.26.1
```

Comment:
not sure why you're upgrading `zerolog`?

Reply:
I think I just wanted to make sure that all Makefile options still work cloning the observer tests from access. I remember I made a go mod tidy. I can remove it from the change.

Reply:
However, it is likely that future changes that include tidy results would do the same.

Reply:
Fixed in: https://github.com/GetElastech/flow-go/commit/5078c8c76b61ec4fb9acf3caefecb365c358b1ba